Tasks/NuGetToolInstaller/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 1,
-        "Patch": 0
+        "Patch": 1
```

Comment:
There were no changes in the NuGetToolInstaller task, so the version shouldn't be revved.